### DISCUSSION TITLE
Can I star or pin a cell in the interactive window?

### DISCUSSION THREAD

**Discussion by kwikwag** [microsoft/vscode-jupyter#13908] [Questions and Answers]
Often in a notebook I perform multiple calculations and then compare outputs on two adjacent cells. My workflow consists of doing many changes to code, often in different cells, evaluating the cells, and then comparing outputs along the way between current and previous results, often in different cells and between rather distant parts in the coding process. In a notebook, I can edit existing cells, as well as to move them up/down, it's easy to compare outputs side-by-side. However, often, because of the distance between operations, often there are many output cells produced between the actions I wish to compare. 

It would be very handy to somehow pin, bookmark or star a cell in the interactive window so that I may go back to it quickly, display only marked cells or clear all unmarked cells. Is there any similar functionality already in place?

**Comment by dglubokov** [comment]
I think it would be really useful. For example, when you work with Python it is very handful to have "import cell" pinned so I can gather all imports in one cell, which is more clear than add imports randomly in different cells or add them every time by up scrolling.

Is it resolved or not?
